src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(stringForConvert) {
+  const result = stringForConvert.trim().split(';');
+
+  const resultObject = result
+    .filter((element) => {
+      if (element.includes(':')) {
+        return element;
+      }
+    })
+    .map((element) => {
+      return element.split(':');
+    })
+    .reduce((object, element) => {
+      return { ...object, [element[0].trim()]: element[1].trim() };
+    }, {});
```

Comment:
The chained methods should have the same indentation to improve code readability. Align the '.filter', '.map', and '.reduce' methods with the '.filter' method.